src/api/@types/fastify/index.d.ts

```diff
@@ -0,0 +1,14 @@
+import fastify from 'fastify';
+import { PgStore } from '../../../pg/pg-store';
+
+declare module 'fastify' {
```

Comment:
Is this approach to typing recommended by Fastify?

Reply:
Sort of... I took that recommendation from here which seems to be a semi-official thing: https://github.com/fastify/fastify/issues/1417#issuecomment-458601746
Happy to hear your thoughts, though, if you think we should do it differently.